test/resource.js

```diff
@@ -1,10 +1,82 @@
 'use strict'
 
 import { assert } from 'chai'
+import jts from 'jsontableschema'
 import Resource from '../src/resource'
 
 describe('Resource', () => {
```

Comment:
Tests look really good but I think we need add some tests with a real data from `data` folder. To be sure our table interface works as expected.